src/Moryx.FactoryMonitor.Endpoints/Models/FactoryStateModel.cs

```diff
@@ -16,10 +16,6 @@ public class FactoryStateModel : VisualizableItemModel
         [DataMember]
         public virtual string BackgroundURL { get; set; }
 
-        [DataMember]
-        [Obsolete]
-        public virtual List<CellModel> Cells { get; set; }
-
```

Comment:
Regenerate API on UI and check usages

Reply:
https://github.com/PHOENIXCONTACT/MORYX-Framework/commit/890dc18bc81fe8a441875394f5df78d1bac0f42f#diff-8c631657b25a84dd7963fe39d2c8c3aaf2404de548e9d3e479e0c7986c6c77aeL13